.github/workflows/sync-staging.yml

```diff
@@ -0,0 +1,53 @@
+# Workflow to sync main branch changes into staging branch.
+# Triggered on pushes to main branch.
+# Jobs:
+# - CI-preSyncCheck: Runs CI checks on main branch.
+# - sync-branch:
+#   - Checks out staging branch
+#   - Merges main into staging with --no-ff
+#   - Pushes merged staging branch
+# Merging with --no-ff creates a merge commit to track changes.
+
+name: Sync Main to Staging
+run-name: Sync-Staging
```

Comment:
I changed the name to something more descriptive in nsc-event let's do the same here.
![image](https://github.com/SeattleColleges/belindas-closet-nestjs/assets/64063020/4db9e497-64cc-434e-befa-4f218f452154)